src/EventStore.Core/Certificates/OptionsCertificateProvider.cs

```diff
@@ -19,6 +19,10 @@ public class OptionsCertificateProvider: CertificateProvider {
 
 			var (certificate, intermediates) = _options.LoadNodeCertificate();
 
+			var certificateCommonName = certificate.GetNameInfo(X509NameType.SimpleName, false);
+			if (certificateCommonName != _options.Certificate.CertificateReservedNodeCommonName) 
+				Log.Error("SSL certificate CN(common name) does not match with CertificateReservedNodeCommonName provided in the EventStoreDB configuration.");
```

Comment:
```suggestion
			var certificateCN = certificate.GetCommonName();
			var reservedNodeCN = _options.Certificate.CertificateReservedNodeCommonName;
			if (certificateCN != reservedNodeCN)
				Log.Warning("Certificate CN: {certificateCN} does not match with the CertificateReservedNodeCommonName configuration setting: {reservedNodeCN}", certificateCN, reservedNodeCN);
```

suggestions:
- change log level to warning
- log the parameters provided